src/org/usfirst/frc/team2585/input/XboxInput.java

```diff
@@ -46,8 +46,17 @@ public boolean boost() {
 	/* (non-Javadoc)
 	 * @see org.usfirst.frc.team2585.input.InputMethod#rotationValue()
 	 */
-	@Override public double rotationValue() {
+	@Override 
+	public double rotationValue() {
 		// Right joystick left/right for rotational movement
 		return controller.getRawAxis(XboxConstants.RIGHT_X_AXIS);
 	}
+	
+	/* (non-Javadoc)
+	 * @see org.usfirst.frc.team2585.input.InputMethod#shoot()
+	 */
+	@Override 
+	public boolean shoot() {
+		return controller.getRawButton(XboxConstants.RIGHT_TRIGGER);
```

Comment:
Technically the right trigger of the Xbox is an analog value from 0 to 1. You can use the right bumper for digital input, however, if you want to continue using right trigger you need to get the analog value and check if it is greater than 0. You will need a deadzone of around 0.15 if you use the triggers.